### DISCUSSION TITLE
ObservableProperty

### DISCUSSION THREAD

**Discussion by TBaltzer** [CommunityToolkit/dotnet#479] [Q&A] [answered]
Is it possible to Name the Observable property to override convention?

[ObservableProperty("FirstName")]
private string firstName;

**Comment by Sergio0694** [comment, marked as answer]
This is not supported by design, no. The property name will automatically match the field. If you want a specific name for the property, just name the field in a way that will result in that name you want to be used for the generated property 🙂
